index.html

```diff
@@ -0,0 +1,20 @@
+    <p id="img-caption">This is an image of Dr Bashir</p>
+    </div>
+    <p id="tribute-info"> this is the paragraph which we need</p>
+    <a id="tribute-link" target="_blank"> This is a link</a>
+</main>
+  
+
+
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+</main>
+<script src="js/index.js"></script>
+</body>
+
+</html>
```

Comment:
-Make sure you configure linters properly :smile:

Reply:
- It would be good to make suggestions and not orders. For example, it would be great if you could configure your linters property so that you could correct some common errors with your code 👍